lumicks/pylake/nb_widgets/correlated_plot.py

```diff
@@ -52,12 +58,16 @@ def plot_correlated(
     aspect_ratio = plot_data.shape[0] / np.max([plot_data.shape])
 
     aspect_ratio = max(0.2, aspect_ratio)
-    fig, (ax1, ax2) = plt.subplots(
-        1,
-        2,
-        figsize=figure_scale * plt.figaspect(aspect_ratio / (aspect_ratio + 1)),
-        gridspec_kw={"width_ratios": [1, 1 / aspect_ratio]},
-    )
+    if vertical:
+        fig, (ax2, ax1) = plt.subplots(2, 1)  # different axis order is on purpose
```

Comment:
it just hit me on seeing the comment here, but should we maybe just rename these to something like `ax_img` and `ax_channel`?

Reply:
Oh man... I can't believe I didn't do this already. It would have saved me some time. :D